### DISCUSSION TITLE
3주차 스터디 회고

### DISCUSSION THREAD

**Discussion by alanhakhyeonsong** [Next-Level-Study/effective-java-study#10] [스터디 회고]
한 줄 회고와 건의사항을 말씀해주세요:)

**Comment by JeongJaew0n** [comment]
책의 설명만 보기보다 예제 코드를 함께 보니 이해가 더 잘됐습니다!
이제부터 이해가 안 될 때는 인텔리제이 켜서 코드를 함께 보는 식으로 공부해야겠습니다.

**Comment by SuJeong68** [comment]
급하게 읽어 많이 부족한 것 같은 한 주 였습니다. 다음 주까지 더 많이 실습해보고 참여하겠습니다 🥲

**Comment by Kurmarkpzgr** [comment]
설계의 방법론적인 이야기에 기술적인 이야기도 많아 이해가 어려웠었습니다.
때문에 실습의 필요성이 많이 느껴졌었습니다.

**Comment by alanhakhyeonsong** [comment]
- 한 줄 회고
  - 이 스터디는 진도를 나가는게 목적이 아니라 **다 함께 하나하나 제대로 알고 넘어가는게 목적**인 스터디니 저 역시도 코드를 많이보고 짜봐야겠습니다.
- 건의사항
  - 분량 조절을 그때그때 잘 해야 할 것 같습니다. 애자일하게... ~~(그렇다고 공부 안해서 그런건 아님)~~
  - 어디서 가져와도 좋으니 예제가 될만한 코드를 좀 더 보도록 합시다.

**Comment by sdh46** [comment]
매일 조금씩 꾸준히 하려고 했는데 좀 바빠서 절반 정도만 그렇게 할 수 있었다. 자투리 시간도 잘 써보려는 노력을 해서 생각하는 시간을 더 늘려보도록 해야겠습니다.

**Comment by opgumo3** [comment]
상속을 잘 안쓰다보니, 이번 장을 읽으면서 글로만 이해하기에는 어려움이 있어서 실습을 해봤는데 그것도 좀 부족했던 것 같습니다.
책에서 얻어갈 수 있는 것들 최대한 많이 얻어가고 싶습니다 😎

**Comment by narangd0** [comment]
발표 설명이 자세해서 책 내용을 한번 더 보는 것 같아 좋았습니다. 굿
스터디를 통해 내용을 복기하며, 접근 제어 매커니즘 - package-private 제한자를 통해 패키지 구조와 설계의 중요성을 느껴 흥미로웠습니다~
그리고 분량 조절 좋았던 듯 굿굿
